phpunit

```diff
@@ -7,8 +7,12 @@ if (!file_exists(__DIR__.'/vendor/symfony/phpunit-bridge/bin/simple-phpunit')) {
     echo "Unable to find the `simple-phpunit` script in `vendor/symfony/phpunit-bridge/bin/`.\nPlease run `composer update` before running this command.\n";
     exit(1);
 }
-if (\PHP_VERSION_ID >= 70000 && !getenv('SYMFONY_PHPUNIT_VERSION')) {
-    putenv('SYMFONY_PHPUNIT_VERSION=6.5');
+if (!getenv('SYMFONY_PHPUNIT_VERSION')) {
+    if (\PHP_VERSION_ID >= 70400) {
+        putenv('SYMFONY_PHPUNIT_VERSION=8.0');
```

Comment:
What about 8.2 here?

Reply:
Why not... https://github.com/sebastianbergmann/phpunit/blob/master/ChangeLog-8.2.md#820---2019-06-07